content/services/index.rst

```diff
@@ -0,0 +1,111 @@
+public: yes
+location: development
+grid_template: true
+headline:
+  - tagline: 'Fully-integrated to achieve your goals'
+    type: 'Services'
+summary: |
+  **OddBird is a full-stack design & development agency**.
+  Our team is fully-integrated,
```

Comment:
T'would be nice to explicitly explain "fully-integrated" here. Maybe "Our team fully integrates UX design and web development -- ready to..."